src/script/page/RightSidebar/GroupParticipantService/GroupParticipantService.tsx

```diff
@@ -69,7 +71,7 @@ const GroupParticipantService: FC<GroupParticipantServiceProps> = ({
   } = useKoSubscribableChildren(activeConversation, ['inTeam', 'isActiveParticipant', 'participating_user_ids']);
   const {teamRole} = useKoSubscribableChildren(selfUser, ['teamRole']);
 
-  const {canChatWithServices, canUpdateGroupParticipants} = generatePermissionHelpers(teamRole);
```

Comment:
can you explain why this isnt working here? just curious

Reply:
Because this is a general permission generator, it's not tied to any particular conversation. We need a permission that is aware of the conversation we are dealing with